Description: Upstream changes introduced in version 2.1.14-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 slurm-llnl (2.1.14-1) unstable; urgency=low
 .
   * New upstream release
 .
 The person named in the Author field signed this changelog entry.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- slurm-llnl-2.1.14.orig/doc/man/man1/sreport.1
+++ slurm-llnl-2.1.14/doc/man/man1/sreport.1
@@ -222,7 +222,9 @@ Valid time formats are...
 HH:MM[:SS] [AM|PM]
 MMDD[YY] or MM/DD[/YY] or MM.DD[.YY]
 MM/DD[/YY]\-HH:MM[:SS]
+.na
 YYYY\-MM\-DD[THH:MM[:SS]]
+.ad
 .TP
 .B Format=<OPT>
 Comma separated list of fields to display in report.
@@ -239,8 +241,9 @@ Period start for report.  Default is 00:
 Valid time formats are...
 HH:MM[:SS] [AM|PM]
 MMDD[YY] or MM/DD[/YY] or MM.DD[.YY]
-MM/DD[/YY]\-HH:MM[:SS]
-YYYY\-MM\-DD[THH:MM[:SS]]
+.na
+MM/DD[/YY]\-HH:MM[:SS] YYYY\-MM\-DD[THH:MM[:SS]]
+.ad
 .RE
 
 .TP
